src/pages/preferences.vue

```diff
@@ -0,0 +1,116 @@
+<template>
+  <VContentPage>
+    <h1>{{ $t('pref-page.title') }}</h1>
+
+    <h2>{{ $t('pref-page.toggle') }}</h2>
+    <ul class="!ps-0">
+      <li
+        v-for="(feature, name) in flags"
+        :key="name"
+        class="flex flex-row items-center"
+      >
+        <VCheckbox
+          :id="name"
+          class="flex-row items-center"
+          :checked="featureState(name) === ON"
+          :disabled="feature.status !== SWITCHABLE"
+          @change="handleChange"
+        >
+          <div>
+            <strong>{{ name }}</strong>
+            <br />
+            {{ feature.description }}
+          </div>
+        </VCheckbox>
+      </li>
+    </ul>
+
+    <p>{{ $t('pref-page.save-desc') }}</p>
+    <VButton variant="primary" @click="handleSave">{{
+      $t('pref-page.save')
+    }}</VButton>
+
+    <h2>{{ $t('pref-page.store-state') }}</h2>
+    <pre><code>{{ flags }}</code></pre>
+
+    <h2>{{ $t('pref-page.content-filtering') }}</h2>
+    <template
+      v-for="featName in ['feat_enabled', 'feat_disabled', 'feat_switchable']"
+    >
+      <template v-for="featState in [ON, OFF]">
+        <i18n
+          v-if="featureState(featName) === featState"
```

Comment:
`v-if` seems to be the best solution for SSR compatibility. Nuxt does not run custom directives like the proposed `v-feat` in SSR, so that HTML is sent to the client and then deleted on client side (which is pretty wasteful).